api/mappers.go

```diff
@@ -59,6 +59,23 @@ func RecordToAPI(record models.Record) server.Record {
 	return response
 }
 
+func JetDropToAPI(jetDrop models.JetDrop) server.JetDrop {
+	pulseNumber := int64(jetDrop.PulseNumber)
+	recordAmount := int64(jetDrop.RecordAmount)
+	result := server.JetDrop{
+		Hash:      NullableString(base64.StdEncoding.EncodeToString(jetDrop.Hash)),
+		JetDropId: NullableString(models.JetDropID(jetDrop.JetID, int64(jetDrop.PulseNumber))),
+		JetId:     NullableString(models.JetIDToString(jetDrop.JetID)),
+		// todo implement this
```

Comment:
could you pls find a task and write down the task number here?

Reply:
fixed